master_startup.py

```diff
@@ -205,20 +207,25 @@ def get_events_list(self):
             # Set full power.
             esc_pwm.ChangeDutyCycle(ESC_MAX_DUTY)
 
-            # TODO: What condition trips this state into LANDED?
-            # landed_time = datetime.datetime.now()
+            # If the acceleration magnitude is next to nothing following the freefall, set state to LANDED
+            if imu_data.get_acc_magnitude() < .1 * 9.81 && imu_data.get_acc_magnitude() > -.1 * 9.81 # Accounting random noise
+            	landed_time = datetime.datetime.now()
+            	current_flight_state = FlightState.LANDED
         elif current_flight_state == FlightState.LANDED:
             # Set zero power.
             esc_pwm.ChangeDutyCycle(ESC_MIN_DUTY)
 
             if datetime.datetime.now() > landed_time + datetime.timedelta(minutes=1):
                 # Begin shutdown procedure
+                cam.stop_recording() 
                 imu_data.add_event("FTS automatic trigger: 10s from launch")
                 log_file.writerow(imu_data.formatted_for_log())
+                time.sleep(10) # Not certain how the module works -- frankly, giving it some time to encode seems sapient
```

Comment:
Pretty quick, so the 10s delay should be extraneous, but a good idea. We'll leave it in for now. Put a TODO in to see if this is necessary.

Reply:
Sounds good. Everything else look fine?

Reply:
(I'll make edits in accordance with these comments before merging, if you're fine with that)